DESCRIPTION

```diff
@@ -44,7 +44,7 @@ Description: Functions to work with date-times and time-spans:
     the 'CCTZ' source code, released under the Apache 2.0 License, are
     included in this package. See <https://github.com/google/cctz> for
     more details.
-License: GPL (>= 2)
```

Comment:
Did you mean to change this?

Reply:
I did, since we were moving to 3, it seemed more precise, but happy to revert.

Reply:
Changed it back, since I kept it elsewhere